cmake/maybe_build_using_clangw.cmake

```diff
@@ -48,8 +48,7 @@ function (maybe_build_using_clangw OE_TARGET)
       -fno-builtin-malloc
       -fno-builtin-calloc
       -fno-builtin
-      -mllvm
-      -x86-speculative-load-hardening)
+      -mspeculative-load-hardening)
```

Comment:
Can we keep this as -mllvm  -x86-speculative-load-hardening?  Will we break code otherwise? If both flags are equivalent can we keep the old one?